src/modules/validation_manager.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include "stdint.h"
+#include "animations/Animation.h"
+
+namespace Animations
+{
+	struct Animation;
+}
+
+namespace Modules
+{
+	/// <summary>
+	/// Manages validation structures and functions including 
+	/// a name blinking animation and the function that plays
+	/// this animation after the system boots.
+	/// </summary>
+	namespace ValidManager
```

Comment:
I realize we have AnimManager but in this specific case I think it's probably best to use the full name ValidationManager for clarity (as "Validation" is the term we've chosen to qualify the process of validating the die in factory)
Also if you'de like you can have `namespace Modules::ValidationManager` for simplicity rather than having `namespace Modules { namespace ValidationManager` (it's a somewhat recent C++ feature)